mmv1/products/cloudbuild/api.yaml

```diff
@@ -230,6 +237,54 @@ objects:
                 exactly_one_of:
                   - github.0.push.0.branch
                   - github.0.push.0.tag
+      - !ruby/object:Api::Type::NestedObject
+        name: 'pubsubConfig'
+        description: |
+          PubsubConfig describes the configuration of a trigger that creates 
+          a build whenever a Pub/Sub message is published.
+        exactly_one_of:
+          - trigger_template
+          - github
+          - pubsub_config
+          - webhook_config
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'subscription'
+            description: |
+              Output only. Name of the subscription.
+          - !ruby/object:Api::Type::String
+            name: 'topic'
+            description: |
+              The name of the topic from which this subscription is receiving messages.
+          - !ruby/object:Api::Type::String
+            name: 'service_account_email'
+            description: |
+              Service account that will make the push request.
+          - !ruby/object:Api::Type::String
+            name: 'state'
```

Comment:
Looks like it should have `output: true`

Reply:
Updated.